src/main/java/fi/aalto/cs/apluscourses/intellij/model/ScalaSdk.java

```diff
@@ -62,6 +62,6 @@ protected ScalaLibraryPropertiesState getPropertiesState(
       @Nullable ScalaLibraryPropertiesState currentState) {
     return new ScalaLibraryPropertiesState(
         ScalaLanguageLevel.findByVersion(scalaVersion).get(),
-        getUris(getCompilerRoots()), new String[0]);
+        getUris(getCompilerRoots()), new String[0], null);
```

Comment:
Passing `compilerBridgeBinaryJar=null` should be fine.

https://github.com/JetBrains/intellij-scala/blob/c24694ef073c4b6573f591e4cb1bcc364f744566/scala/scala-impl/src/org/jetbrains/plugins/scala/project/ScalaLibraryProperties.scala#L15C1-L18C86
```java
/**
[...]
 * @param _compilerBridgeBinaryJar optional compiler bridge jar.<br>
 *                                 When it's None, a bundled bridge will be used (see `Scala/lib/jps` directory in Scala plugin distribution).<br>
 *                                 Custom, non-bundled bridge is mostly required to be able to compile code
 *                                 with RC/Nightly versions of new Scala 3.x compiler
 */
```

About compiler bridge: https://www.scala-sbt.org/1.x/docs/Compiler-Interface.html